packages/contracts/test/ProxyBorrowerWrappersScript.js

```diff
@@ -96,7 +96,7 @@ contract('BorrowerWrappers', async accounts => {
     zeroStaking = ZEROContracts.zeroStaking
     zeroToken = ZEROContracts.zeroToken
     wrbtcToken = contracts.wrbtcTokenTester
-    sovFeeCollector = ZEROContracts.mockFeeSharingProxy.address
+    sovFeeCollector = ZEROContracts.mockFeeSharingCollector.address
```

Comment:
rename all sovFeeCollector => feeSharingCollector  
else it is misleading and is easily confused with SOV collector

Reply:
Updated in this [commit](https://github.com/DistributedCollective/zero/pull/174/commits/00688cd15e330f453d4ffa48d0f77477b97bcae6)